lib/view/fiscaat/saldo/SaldiSommenResponseView.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace CsrDelft\view\fiscaat\saldo;
+
+
+use CsrDelft\view\SmartyTemplateView;
+use DateTime;
+
+class SaldiSommenResponseView extends SmartyTemplateView {
```

Comment:
Graag een docblock met `@property CiviSaldoModel $model`, dan weet je wat model in deze view betekend en gaat highlighting werken.

Reply:
En een `@author` en `@date`